stonesoup/platform/base.py

```diff
@@ -189,3 +192,17 @@ class MovingPlatform(Platform):
 
 class MultiTransitionMovingPlatform(Platform):
     _default_movable_class = MultiTransitionMovable
+
+
+class PathBasedPlatform(MovingPlatform):
+    path: GroundTruthPath = Property(doc="Ground truth path used for the platform. "
+                                     "This is for use in cases when platform paths are imported "
+                                     "from external data, e.g., from a CSV file")
+
```

Comment:
Is it worth giving the `transition_model` property a default value of `None` as it is no longer required required?

Reply:
Added.